app/views/rosters/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<% view = FormView.new(subject: @roster) %>
+
+<div class="site-content">
+  <div class="site-content-cap">
+    <h2 class="site-content-heading">Add students</h2>
+  </div>
+  <div class="site-content-body">
+    <div class="markdown-body">
+      <p> Next, let's set up a "roster", so you can easily track student progress on your dashboard.
+      <p> To create your roster, you'll need to paste in unqiue identifiers, one per line. These identifiers can be:</p>
+
+      <ul>
+        <li>Student last names</li>
+        <li>Student email addresses</li>
+        <li>Any other unique piece of information</li>
+      </ul>
+
+
+      <p>You can always come back and complete this step later.</p>
+    </div>
+    </br>
+    <div>
+      <%= form_tag roster_path(@organization), method: :post do |f| %>
+        <dl class="<%= view.form_class_for(:identifiers) %> short-field">
+          <dt><label>Paste your list of identifiers below (one per line, please!):</label></dt>
+          <dd><%= text_area_tag(:identifiers, '', class: 'form-control input-block') %></dd>
+          <%= render('shared/form_error', errors: view.error_message_for(:identifiers)) if view.errors_for?(:identifiers) %>
+        </dl>
+
+        <dl class="<%= view.form_class_for(:identifier_name) %> short-field">
+          <dt><label>Give this batch of identifiers a name (in case you'd like to re-use it):</label></dt>
```

Comment:
This "name" isn't actually a unique identifier or a way to identify a roster. It's a human-readable description of what the identifiers are. E.g. "Student email addresses" or "UCSF student number"

@mozzadrella